components/gateway/internal/gqlschema/schema.graphql

```diff
@@ -51,8 +51,9 @@ type Application {
     status: ApplicationStatus!
     webhooks: [ApplicationWebhook!]!
     healthCheckURL: String
-    apis: [APIDefinition!]!
-    eventAPIs: [EventAPIDefinition!]!
+    """ groupName allows to find different versions of the same API """
```

Comment:
The parameter name should be equal to the property in `APIDefinition`, so in this case: `group`. However, I don't like the name, as it sounds like it could contain different APIs, not just the same APIs with different versions.